### PR TITLE
Getting started data prepper

### LINKED ISSUE #1457: Get Started with Data Prepper
Migrate over https://github.com/opensearch-project/data-prepper/blob/main/docs/getting_started.md

### PR DESCRIPTION
**Pull request by carolxob**
### Description
Merged content from Data Prepper repo documentation into existing page on OpenSearch documentation website. 

### Issues Resolved
Fixes #1457 

### Checklist
- [ x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by carolxob** [comment]
@dlvenable This is ready for your review.

File: _clients/data-prepper/get-started.md [hunk 1]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+
+You can pull the Docker image:
+
+```
 docker pull opensearchproject/data-prepper:latest
 ```
 
-## 2. Define a pipeline
+If you have special requirements that require you build from source, or if you
+want to contribute, please see the [Developer Guide](developer_guide.md).
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:31]: This Developer Guide is located in the Data Prepper GitHub repository. So we should link to it using a fully-qualified URL:

https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md

File: _clients/data-prepper/get-started.md [hunk 2]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+
+You must configure Data Prepper with a pipeline before running it.
+
+You will configure two files:
+
+* `data-prepper-config.yaml`
+* `pipelines.yaml`
+
+Depending on what you want to do, we have a few different guides to configuring Data Prepper.
+* [Trace Analytics](trace_analytics.md) - Learn how to setup Data Prepper for trace observability
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:43]: These three files are also in the GitHub repository and need fully qualified links:

Basically add the following in front of the links:
`https://github.com/opensearch-project/data-prepper/tree/main/docs/`

File: _clients/data-prepper/get-started.md [hunk 3]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+For Data Prepper 2.0 or above, use this command:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+For Data Prepper before version 2.0:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:latest
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:106]: Data Prepper `latest` could be 2.0 soon. Perhaps we should change this to `data-prepper:1` which loads the latest from the 1.x series.

File: _clients/data-prepper/get-started.md [hunk 4]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file is read from `config/log4j2.properties` in the application's home directory. 
+By default, it's using `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper before version 2.0, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would 
+like to pass a custom log4j2 properties file. If no properties file is provided, Data Prepper will default to the log4j2.properties file in the *shared-config* directory.
+
+## Next Steps
+
+All Data Prepper instances expose a few APIs. The [API documentation](core_apis.md) outlines these APIs and
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:124]: Another link that needs to be fully qualified.

File: _clients/data-prepper/get-started.md [hunk 5]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+how to configure the server.
+
+Trace Analytics is an important Data Prepper use case. If you haven't yet configure it,
+please visit the [Trace Analytics documentation](trace_analytics.md).
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:128]: broken link

File: _clients/data-prepper/get-started.md [hunk 6]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+Log Ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation](log_analytics.md).
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:130]: broken link

File: _clients/data-prepper/get-started.md [hunk 7]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+To run Data Prepper with a Logstash configuration, please visit the [Logstash Migration Guide](logstash_migration_guide.md).
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:132]: broken link

File: _clients/data-prepper/get-started.md [hunk 8]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+To monitor Data Prepper, please read the [Monitoring](monitoring.md) page.
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:134]: broken link

File: _clients/data-prepper/get-started.md [hunk 9]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+If you are migrating from Open Distro Data Prepper, visit the [Migrating from Open Distro](migrating_from_opendistro.md) page. Otherwise, please continue.
 
-To use the Docker image, pull it like any other image:
+## 1. Installing Data Prepper
 
-```bash
+There are two ways to install Data Prepper for running:
+
+1. Run the Docker image
+2. Build from source
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:19]: It is possible to install from `tar.gz` now. (Our repo is out-of-date on this).

I'm not sure we want to have "Build from source" in the opensearch.org documentation. It is mostly useful in certain situations (bleeding-edge, contributions).

**Review by dlvenable — CHANGES_REQUESTED** [review]
Thanks for making these improvements to the opensearch.org documentation!

File: _clients/data-prepper/get-started.md (hunk 6 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:130]: @carolxob: Let's set these up in such a way that they'll match the new markdown files once we migrate them over.

For example:

```suggestion
Log Ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics.md).
```

File: _clients/data-prepper/get-started.md (hunk 9 above)
Comment (carolxob) [reply _clients/data-prepper/get-started.md:19]: @dlvenable Do you have a link to the tar.gz file?

File: _clients/data-prepper/get-started.md (hunk 2 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:43]: Switch to: `{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/`

File: _clients/data-prepper/get-started.md (hunk 4 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:124]: Switch to: `{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/api/`

File: _clients/data-prepper/get-started.md (hunk 5 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:128]: Switch to: {{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/

File: _clients/data-prepper/get-started.md [hunk 10]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
 e51e700e-5cab-4f6d-879a-1c3235a77d18
 b4ed2d7e-cf9c-4e9d-967c-b18e8af35c90
 ```
+The remainder of this page shows examples for running from the Docker image. If you
+built from source, refer to [Developer Guide](developer_guide.md) for more information.
```
Comment (Naarcha-AWS) [inline _clients/data-prepper/get-started.md:92]:
```suggestion
built from source, refer to [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md) for more information.
```

File: _clients/data-prepper/get-started.md (hunk 7 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:132]:
```suggestion
To run Data Prepper with a Logstash configuration, please visit the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
```

File: _clients/data-prepper/get-started.md (hunk 8 above)
Comment (Naarcha-AWS) [inline _clients/data-prepper/get-started.md:134]:
```suggestion
To monitor Data Prepper, please read the [Monitoring]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/monitoring/) page.
```

**Review by dlvenable — CHANGES_REQUESTED** [review]
Some of the URLs will still be dead links as the content has not been copied over.

The full list of available pages is found here:

https://github.com/opensearch-project/documentation-website/tree/main/_clients/data-prepper

You can see that pages like `trace-analytics` or `migrate-open-distro` are not present.

Those files are available in Data Prepper documentation and you can see there all at the following link:

https://github.com/opensearch-project/data-prepper/tree/main/docs

File: _clients/data-prepper/get-started.md (hunk 9 above)
Comment (dlvenable) [reply _clients/data-prepper/get-started.md:19]: The tar.gz changes with every build. I think the best solution is to point the customer to the download page: https://opensearch.org/downloads.html#data-prepper

That will have the latest version available.

Comment (carolxob) [reply _clients/data-prepper/get-started.md:19]: @Naarcha-AWS FYI, there is some new information here.

Comment (carolxob) [reply _clients/data-prepper/get-started.md:19]: @dlvenable Noted - I am going to make a separate issue to address the tar.gz installation instructions.

**Review by Naarcha-AWS — APPROVED** [review]

File: _clients/data-prepper/get-started.md [hunk 11]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+
+You must configure Data Prepper with a pipeline before running it.
+
+You will configure two files:
+
+* `data-prepper-config.yaml`
+* `pipelines.yaml`
+
+Depending on what you want to do, we have a few different guides to configuring Data Prepper.
+* [Trace Analytics]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/) - Learn how to setup Data Prepper for trace observability
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:43]: @Naarcha-AWS , I still don't see these available in the documentation. I expect that these links will be broken.

File: _clients/data-prepper/get-started.md [hunk 12]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+how to configure the server.
+
+Trace Analytics is an important Data Prepper use case. If you haven't yet configure it,
+please visit the [Trace Analytics documentation]{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/).
+
+Log Ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
+
+To run Data Prepper with a Logstash configuration, please visit the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
+
+To monitor Data Prepper, please read the [Monitoring]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/monitoring/) page.
```
Comment (dlvenable) [inline _clients/data-prepper/get-started.md:134]: @Naarcha-AWS , This also does not appear to have been copied over. So another likely broken link.

**Review by dlvenable — CHANGES_REQUESTED** [review]
I still believe there are broken links.

File: _clients/data-prepper/get-started.md [hunk 13]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+For Data Prepper before version 2.0, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would 
+like to pass a custom log4j2 properties file. If no properties file is provided, Data Prepper will default to the log4j2.properties file in the *shared-config* directory.
+
+## Next Steps
+
+All Data Prepper instances expose a few APIs. The [API documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/api/) outlines these APIs and
+how to configure the server.
+
+Trace Analytics is an important Data Prepper use case. If you haven't yet configure it,
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:127]:
```suggestion
Trace Analytics is an important Data Prepper use case. If you haven't yet configured it,
```

File: _clients/data-prepper/get-started.md [hunk 14]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+please visit the [Trace Analytics documentation]{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/).
+
+Log Ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:130]: Same. Unless "Log Ingestion" is a proper noun, "ingestion" should be lowercase.

Comment (natebower) [inline _clients/data-prepper/get-started.md:130]:
```suggestion
Log Ingestion is also an important Data Prepper use case. To learn more, see the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
```

File: _clients/data-prepper/get-started.md [hunk 15]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+To run Data Prepper with a Logstash configuration, please visit the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:132]:
```suggestion
To learn how to run Data Prepper with a Logstash configuration, see the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
```

File: _clients/data-prepper/get-started.md (hunk 12 above)
Comment (natebower) [inline _clients/data-prepper/get-started.md:134]:
```suggestion
For information on how to monitor Data Prepper, see the [Monitoring]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/monitoring/) page.
```

File: _clients/data-prepper/get-started.md [hunk 16]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+
+You can pull the Docker image:
+
+```
 docker pull opensearchproject/data-prepper:latest
 ```
 
-## 2. Define a pipeline
+If you have special requirements that require you build from source, or if you
+want to contribute, please see the [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md).
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:31]:
```suggestion
want to contribute, see the [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md).
```

File: _clients/data-prepper/get-started.md [hunk 17]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+Trace Analytics is an important Data Prepper use case. If you haven't yet configure it,
+please visit the [Trace Analytics documentation]{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/).
+
+Log Ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
+
+To run Data Prepper with a Logstash configuration, please visit the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
+
+To monitor Data Prepper, please read the [Monitoring]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/monitoring/) page.
+
+## Other Examples
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:136]:
```suggestion
## Other examples
```

File: _clients/data-prepper/get-started.md [hunk 18]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+We have other several Docker [examples](https://github.com/opensearch-project/data-prepper/tree/main/examples/)
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:138]:
```suggestion
We have several other Docker [examples](https://github.com/opensearch-project/data-prepper/tree/main/examples/)
```

File: _clients/data-prepper/get-started.md [hunk 19]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+that allow you to run Data Prepper in different scenarios.
```
Comment (natebower) [inline _clients/data-prepper/get-started.md:139]: I would rephrase this as "For other Docker examples that allow you to run Data Prepper in different scenarios, see [link]."

**Review by natebower — COMMENTED** [review]
@carolxob Please see my comments and changes and let me know if you have any questions. Thanks!

File: _clients/data-prepper/get-started.md [hunk 20]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+However you configure your pipeline, you will run Data Prepper the same way. You run the Docker
+image and supply both the `pipelines.yaml` and `data-prepper-config.yaml` files.
+
+For Data Prepper 2.0 or above, use this command:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+For Data Prepper before version 2.0:
```
[the remark this answers is not in the document]
Comment (dlvenable) [reply _clients/data-prepper/get-started.md:103]: It would be 1.5 or earlier.

File: _clients/data-prepper/get-started.md [hunk 21]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+
+```
+curl -X POST http://localhost:4900/shutdown
+```
+### Additional Configurations
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file is read from `config/log4j2.properties` in the application's home directory. 
+By default, it's using `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper before version 2.0, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would
```
[the remark this answers is not in the document]
Comment (dlvenable) [reply _clients/data-prepper/get-started.md:119]: Also, 1.5

File: _clients/data-prepper/get-started.md [hunk 22]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+## 2. Configuring Data Prepper
+
+You must configure Data Prepper with a pipeline before running it.
+
+You will configure two files:
+
+* `data-prepper-config.yaml`
+* `pipelines.yaml`
+
+Depending on what you want to do, we have a few different guides to configuring Data Prepper.
```
Comment (carolxob) [inline _clients/data-prepper/get-started.md:42]:
```suggestion
Depending on your use case, we have a few different guides to configuring Data Prepper.
```

File: _clients/data-prepper/get-started.md [hunk 23]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
 e51e700e-5cab-4f6d-879a-1c3235a77d18
 b4ed2d7e-cf9c-4e9d-967c-b18e8af35c90
 ```
+The remainder of this page shows examples for running from the Docker image. If you
+built from source, refer to [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md) for more information.
+
+However you configure your pipeline, you will run Data Prepper the same way. You run the Docker
+image and supply both the `pipelines.yaml` and `data-prepper-config.yaml` files.
+
+For Data Prepper 2.0 or above, use this command:
```
Comment (carolxob) [inline _clients/data-prepper/get-started.md:97]:
```suggestion
For Data Prepper 2.0 or later, use this command:
```

File: _clients/data-prepper/get-started.md (hunk 20 above)
Comment (carolxob) [inline _clients/data-prepper/get-started.md:103]:
```suggestion
For Data Prepper version 1.5 or earlier:
```

File: _clients/data-prepper/get-started.md [hunk 24]
```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
...
+```
+
+Once Data Prepper is running, it will process data until it is shutdown. Once you are done, shut it down with
+
+```
+curl -X POST http://localhost:4900/shutdown
+```
+### Additional Configurations
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file is read from `config/log4j2.properties` in the application's home directory.
```
Comment (carolxob) [inline _clients/data-prepper/get-started.md:116]:
```suggestion
For Data Prepper 2.0 or later, the Log4j 2 configuration file is read from `config/log4j2.properties` in the application's home directory. 
```

File: _clients/data-prepper/get-started.md (hunk 21 above)
Comment (carolxob) [inline _clients/data-prepper/get-started.md:119]:
```suggestion
For Data Prepper version 1.5 or earlier, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would 
```

File: _clients/data-prepper/get-started.md (hunk 13 above)
Comment (carolxob) [inline _clients/data-prepper/get-started.md:127]:
```suggestion
Trace analytics is an important Data Prepper use case. If you haven't yet configured it,
```

File: _clients/data-prepper/get-started.md (hunk 14 above)
Comment (carolxob) [inline _clients/data-prepper/get-started.md:130]:
```suggestion
Log ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
```

File: _clients/data-prepper/get-started.md (hunk 18 above)
Comment (carolxob) [inline _clients/data-prepper/get-started.md:138]:
```suggestion
For other Docker examples that allow you to run Data Prepper in different scenarios, see [examples] (https://github.com/opensearch-project/data-prepper/tree/main/examples/) 
```

**Review by carolxob — COMMENTED** [review]
Incorporated editorial review changes.

File: _clients/data-prepper/get-started.md (hunk 11 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:43]: @dlvenable: I went ahead and made issue to migrate Trace Analytics before we merge this page. See issue #1837

File: _clients/data-prepper/get-started.md (hunk 12 above)
Comment (Naarcha-AWS) [reply _clients/data-prepper/get-started.md:134]: I went ahead and made an issue to migrate the monitoring page. see #1838

File: _clients/data-prepper/get-started.md [hunk 25]
```diff
@@ -9,15 +9,42 @@ nav_order: 1
...
+## 2. Configuring Data Prepper
+
+You must configure Data Prepper with a pipeline before running it.
+
+You will configure two files:
+
+* `data-prepper-config.yaml`
+* `pipelines.yaml`
+
+Depending on what you want to do, we have a few different guides to configuring Data Prepper.
```
Comment (carolxob) [inline _clients/data-prepper/get-started.md:42]:
```suggestion
Depending on your use case, we have a few different guides to configuring Data Prepper.
```

File: _clients/data-prepper/get-started.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (carolxob) [reply _clients/data-prepper/get-started.md:119]: @dlvenable Is this before 2.0? or 1.5?
